### ISSUE TITLE
Firefox: Overflow menu icon in button groups is smaller than the rest of the group

### ISSUE THREAD

**Issue by DrnglVrgs** [salesforce-ux/design-system#547]
# Description
A bug where the icon button is smaller than the rest of the group when using button groups with an overflow menu icon

# Screenshot
![screenshot 2018-05-08 16-23-41](https://user-images.githubusercontent.com/27937411/39778374-7eb9c728-52dc-11e8-9e7f-234e45ac0c71.png)

# Enviroment
I found this bug in Firefox 52.7.3
I tested it in Chromium 66.0.3359.117 and did not encounter it

# Version
Found it in [SLDS 2.6.0](https://www.lightningdesignsystem.com/components/button-groups/) (Summer '18), checked the archives and the bug is present at least since [SLDS 2.4.5 ](https://archive-2_4_5.lightningdesignsystem.com/components/button-groups)(Winter '18) and it is not present in [SLDS 2.3.1](https://archive-2_3_1.lightningdesignsystem.com/components/button-groups) (Summer '17)

# Steps
Using firefox access the [docs](https://www.lightningdesignsystem.com/components/button-groups/) and select "Overflow Menu Icon" in examples

**Comment by engai** [comment]
Hi @DrnglVrgs, thank you for bringing this to our attention. We'll look into it.

**Comment by engai** [comment]
@DrnglVrgs We're only able to replicate the issue when zoomed at 90%, is your browser zoomed in at 90%? It may be a minor issue with how Firefox scales.

**Comment by DrnglVrgs** [comment]
I have Firefox in 100% zoom and have the problem, but if I zoom to 110% no more problem. 
It may be due to the size of the font, I usually have size 13, when i change it to 15 or more the problem disappears

**Comment by engai** [comment]
@DrnglVrgs yeah, it seems to just be a browser specific font issue that doesn't quite have a good fix on our end. 

Let me know if you have any further thoughts on this.

**Comment by engai** [comment]
Closing this issue as there doesn't seem to be more of a Firefox rendering bug. Feel free to comment/reopen if needed.
